br/pkg/lightning/backend/kv/sql2kv.go

```diff
@@ -433,7 +434,7 @@ func (kvcodec *tableKVEncoder) Encode(
 	}
 	kvPairs := kvcodec.se.takeKvPairs()
 	for i := 0; i < len(kvPairs.pairs); i++ {
-		kvPairs.pairs[i].RowID = rowID
+		kvPairs.pairs[i].RowID = kv.IntHandle(rowID).Encoded()
```

Comment:
maybe directly encode a [8]byte from int64. This is a hot path, performance may matter

Reply:
Updated.